src/com/company/QueueClassConcurrent.java

```diff
@@ -0,0 +1,31 @@
+package com.company;
+
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+public class QueueClassConcurrent implements Runnable {
+
+    private Queue<Integer> normalQueue;
+    private ConcurrentLinkedQueue<Integer> concurrentSafeQueue;
+    private int counter;
+
+    public QueueClassConcurrent(Queue<Integer> normalQueue, ConcurrentLinkedQueue<Integer> concurrentSafeQueue, int counter) {
+        this.normalQueue = normalQueue;
+        this.concurrentSafeQueue = concurrentSafeQueue;
+        this.counter = counter;
+    }
+
+    @Override
+    public void run() {
+        System.out.println(Thread.currentThread().getName() + " started");
+        for (int i = 0; i < counter; i++) {
+                try{
+                    normalQueue.offer(i);
+                } catch (NullPointerException e){
```

Comment:
NullPointerException is a uncheacked exception. Why do you use try-catch block here?